packages/esm-service-queues-app/src/transition-latest-queue-entry/transition-overflow-menu-item/transition-overflow-menu-item.component.tsx

```diff
@@ -0,0 +1,36 @@
+import { SkeletonText, OverflowMenuItem } from '@carbon/react';
+import { showModal, useVisit } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface TransitionOverflowMenuItemProps {
+  patientUuid: string;
+}
+
+const TransitionOverflowMenuItem: React.FC<TransitionOverflowMenuItemProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const { activeVisit, isLoading } = useVisit(patientUuid) || {};
+
+  const handleLaunchModal = () => {
+    const dispose = showModal('transition-patient-to-latest-queue-modal', {
```

Comment:
The modal needs to be registered in `routes.json`. Please refer the docs on how to register the modal - https://o3-docs.openmrs.org/docs/modal-system